### ISSUE TITLE
Co2 emissions factors for electic trains 

### ISSUE THREAD

**Issue by redfrexx** [pledge4future/co2calculator#105]
For electric long distance trains there are two co2 factors (ids 65 and 69). One from UBA and one from Öko-Institut. Which one should be used? 

The data was selected based on these parameters. 

```python
  parameters = {"subcategory": TransportationMode.TRAIN,
                "fuel_type": TrainFuel.ELECTRIC,
                "range": BusTrainRange.LONG_DISTANCE}
```

#### Returned data: 

65,public_transport,train,UBA,tremod,Personennahverkehrszug,P.km,average,,,local,electric,kg/P.km,0.0524,,
69,public_transport,train,Öko-Institut,gemis,Zug-Personen-Fern-Elektro-DE-2020-Basis,P.km,,750,,long-distance,electric,kg/P.km,0.00948,,

**Comment by han16nah** [comment]
another similar case:

67,transport,train,UBA,tremod,Strassen-Stadt-U-Bahn,P.km,average,,,,electric,kg/P.km,0.0548,,
102,transport,tram,UBA,,Strassen-/Stadt-/U-Bahn,P.km,,,,,,kg/P.km,0.078,,

**Comment by redfrexx** [comment]
For trains with fuel_type=diesel there also seem to be two co2 factors.

**Comment by veitu** [comment]
I'm not sure which CO2 factors to choose for the trains. The ones from gemis have the advantages that they are for 2020 while the ones from tremod are from 2008. The ones from tremod have the advantages that they also have an average co2 factor if the fuel type of the train is unknown, and our co2 factors for cars and buses are also from tremod, so it would be more consistent, I guess.

Regarding the Straßenbahn duplicate: As far as I can tell, the co2 factor from line 102 is not necessary, so I removed it. The one from line 67 is used in the commuting function.

**Comment by han16nah** [comment]
Hm, shall we go for Tremod?
According to ifeu, there should be a recent version 6.0 for 2020: https://www.ifeu.de/methoden-tools/modelle/tremod/
But I can not find a link directly to the program, it does not seem to open (except for what is also in Probas), right?

We could also have a look at [HBEFA](https://www.hbefa.net/d/index.html)

**Comment by veitu** [comment]
Sure, let's go for Tremod! I also can't find any link to the Tremod program itself, we would probably have to order it or wait until they update the emission factors in Probas...

**Comment by han16nah** [comment]
Future task to contact someone regarding TREMOD
For now, we deleted the ones from GEMIS, to be merged soon #107
